src/main/webapp/app/exam/participate/information/exam-information.component.html

```diff
@@ -22,9 +22,12 @@
                 </span>
             </ng-template>
         </ng-container>
-        <span class="mx-2" *ngIf="studentExam && studentExam.workingTime">
-            <strong class="font-weight-bold">{{ 'artemisApp.exam.durationWithFormat' | artemisTranslate }}:</strong>
-            {{ studentExam.workingTime || 0 | artemisDurationFromSeconds }}
+        <span class="mx-2" *ngIf="exam && exam.startDate && exam.endDate">
+            <strong class="font-weight-bold">{{ 'artemisApp.exam.duration' | artemisTranslate }}:</strong>
+            {{ normalWorkingTime() || 0 | artemisDurationFromSeconds }}
```

Comment:
We try to avoid method calls from the template (as they are potentially executed every render frame). These values don't seem to change once an exam is passed to the component, so can we calculate the values once and then reference fields here?

Reply:
Actually we can probably ignore this since the pipeline runs over the return value.